maploc/data/dataset.py

```diff
@@ -126,44 +127,74 @@ def get_view(self, idx, scene, seq, name, seed, bbox_tile):
         cam_dict = self.data["cameras"][scene][seq][self.data["camera_id"][idx]]
         cam = Camera.from_dict(cam_dict).float()
 
-        if "roll_pitch_yaw" in self.data:
-            roll, pitch, yaw = self.data["roll_pitch_yaw"][idx].numpy()
```

Comment:
Some older datasets might still have the `roll_pitch_yaw` entry instead of `R_c2w` - can we ensure backward compatibility?

Reply:
For backward compatibility, I've created a function `compose_rotmat(roll, pitch, yaw)` that returns `R_c2w`.

However, this assumes that when a dataset has `roll_pitch_yaw`, these Euler angles represent the same angles as those obtained using `decompose_rotmat(R_c2w)` for the other datasets. Is that the case?

Reply:
For backward compatibility, I've created a function `compose_rotmat(roll, pitch, yaw)` that returns `R_c2w` [in this commit](https://github.com/facebookresearch/OrienterNet/pull/49/commits/fcb928ba78e17acd10a51f17d7d7fe146c19d0bb
).

However, this assumes that when a dataset has `roll_pitch_yaw`, these Euler angles represent the same angles as those obtained using `decompose_rotmat(R_c2w)` for the other datasets. Is that the case?

Reply:
Nice! Yes this should be.